tests/dogtag/pytest-ansible/pytest/performance_test/test_cert_enrollment.py

```diff
@@ -74,7 +79,8 @@ def __init__(self, sn_uid):
         for enrollment_result in enrollment_results:
             request_data = enrollment_result.request
```

Comment:
We should verify that `request_data.request_status == 'complete'`.

Reply:
Done

Reply:
Added